torch/sparse/_triton_ops_meta.py

```diff
@@ -732,7 +737,11 @@ def main(op="scatter_mm", force=False, dtype=torch.float16, verbose=True):
     sizes3_lst = [3 * sz for sz in [64, 128] + sizes_lst if sz <= 2048]
     shapes_lst = [(sz, sz) for sz in sizes_lst[:-4] + sizes3_lst]
     shapes_lst.extend([(3072, 768), (768, 3072)])
-    blocksize_lst = [(16, 16), (32, 32), (64, 64), (128, 128)]
+    if dtype is torch.int8:
+        # triton does not support smaller blocks than 32
+        blocksize_lst = [(32, 32), (64, 64), (128, 128)]
```

Comment:
It might be worth it to try going to blocksize 256 for int8? With an 8bit element 256*256 blocks would be the same data size as 128*128 blocks and a 32bit element.

Reply:
Absolutely. This PR enables the int8 support and a follow-up PR ought to provide tuned parameters for A100.

Reply:
Resolving via https://github.com/pytorch/pytorch/pull/134035